.changeset/short-dots-shake.md

```diff
@@ -2,4 +2,4 @@
 '@razorpay/blade': minor
 ---
 
-feat(blade): add filter chip date picker
+feat(DatePicker): add `FilterChipDatePicker` component
```

Comment:
2 changeset files?

Reply:
these change sets were for 2 components 😢 
filterChipSelectInput ,FilterChipDatePicker